napari/__main__.py

```diff
@@ -288,6 +292,12 @@ def _run():
         # it will collect it and hang napari at start time.
         # in a way that is machine, os, time (and likely weather dependant).
         viewer = Viewer()
+
+        # For backwards compatibility
+        # If the --stack option is provided without additional arguments
+        # just set stack to True similar to the previous store_true action
+        if args.stack and len(args.stack) == 1 and len(args.stack[0]) == 0:
+            args.stack = True
```

Comment:
maybe we would like to add here a deprecation warning?

Reply:
Yes, I think we should drop the pure `--stack` call... But maybe I'm missing why we're keeping the old `--stack` format? It should be entirely encompassed by just putting `--stack` in the right spot in the command.

Reply:
It may be nice to have one version with transition, to not need to hack error message and not need to respond to multiple questions about why this command stop working.

Reply:
I added a deprecation warning, which I basically copied from `build/lib/napari/layers/utils/layer_utils.py`.